fml/file.h

```diff
@@ -124,6 +124,10 @@ bool VisitFiles(const fml::UniqueFD& directory, const FileVisitor& visitor);
 bool VisitFilesRecursively(const fml::UniqueFD& directory,
                            const FileVisitor& visitor);
 
+/// Helper method to recursively remove files and subdirectories inside the
+/// directory. The directory itself will not be removed.
+void RemoveFilesInDirectory(const fml::UniqueFD& directory);
```

Comment:
Some error handling would be good here. Return a bool to notify that the files were actually deleted maybe. Right now, failures are silent.

Reply:
Done.